test/http-test.js

```diff
@@ -11,30 +11,38 @@ const Outpoint = require('../lib/primitives/outpoint');
 const MTX = require('../lib/primitives/mtx');
 const FullNode = require('../lib/node/fullnode');
 const pkg = require('../lib/pkg');
-const Network = require('../lib/protocol/network');
-const network = Network.get('regtest');
 
 if (process.browser)
   return;
 
+const ports = {
+  p2p: 49331,
```

Comment:
Don't we usually call this `public` elsewhere?

Reply:
Possibly, however it's more clear here as the peer-to-peer port, in this case it's not public as it's only for the test.